src/components/ebay-icon-button/icon-button.stories.js

```diff
@@ -0,0 +1,118 @@
+import button from './index.marko';
+import renderBody from './examples/body.marko';
+import readme from './README.md';
+
+const Template = (args) => ({
+    input: {
+        ...args,
+        renderBody,
+    },
+});
+// const Template = args =({ input: withRenderBody(args) })
+
+export default {
+    title: 'ebay-icon-button',
+    component: button,
+    parameters: {
+        docs: {
+            description: {
+                component: readme,
+            },
+        },
+    },
+    argTypes: {
+        renderBody: {},
+        href: {
+            description: 'for link that looks like a button',
+        },
+        disabled: {
+            description: '',
+            table: {
+                category: 'Toggles',
+                defaultValue: {
+                    summary: 'false',
+                },
+            },
+        },
+        'partially-disabled': {
+            description: 'disables events, but does not show button as disabled',
+            table: {
+                defaultValue: {
+                    summary: 'false',
+                },
+                category: 'Toggles',
+            },
+        },
+        'badge-number': {
+            description: 'used as the number to be placed in the badge',
+            table: {
+                category: 'Badge (only with variant=icon)',
+            },
+            type: 'number',
+        },
+        'badge-aira-label': {
+            description: 'passed as the `aria-label` directly to the badge',
+            table: {
+                category: 'Badge (only with variant=icon)',
```

Comment:
Do we still have `variant`? Or is it now implied when badge attributes are used?